io.github.NickKarpowicz.LightwaveExplorer.json

```diff
@@ -0,0 +1,145 @@
+                            "rm -f /app/opt/intel/oneapi/mkl/latest/lib/intel64/*.so*",
+                            "rm -rf /app/opt/intel/oneapi/conda_channel"],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://registrationcenter-download.intel.com/akdlm/IRC_NAS/992857b9-624c-45de-9701-f6445d845359/l_BaseKit_p_2023.2.0.49397_offline.sh",
+                    "sha256": "3dd04d2cc9ba2a8785580ba09068dbaf9aff9aa930b5fa7f10688210ea9be4a0"
+        
+                }
+            ]
+        },
+        {
+            "name": "fmt",
+            "config-opts": ["-DCMAKE_POSITION_INDEPENDENT_CODE=TRUE", "-DFMT_TEST=OFF"],
+            "buildsystem": "cmake-ninja",
+            "sources": [ 
+                { 
+                    "type": "git", 
+                    "url": "https://github.com/fmtlib/fmt", 
+                    "tag": "10.1.0",
+                    "commit": "e57ca2e3685b160617d3d95fcd9e789c4e06ca88" } 
+                ]
+        },
+        {
+            "name": "gcem",
+            "buildsystem": "cmake-ninja",
+            "sources": [ 
+                { 
+                    "type": "git", 
+                    "url": "https://github.com/kthohr/gcem", 
+                    "tag": "v1.17.0",
+                    "commit": "8422f5307b0498d09cf626f38acb03fe9f1efd94" } 
+                ]
+        },
+        {
+            "name": "dlib",
+            "buildsystem": "simple",
+            "build-commands":["cp -R dlib /app/include/"],
+            "sources": [ 
+                { 
+                    "type": "git", 
+                    "url": "https://github.com/davisking/dlib", 
+                    "tag": "v19.24.2",
+                    "commit": "39749fba5163d4e25030358f6aa8d91617bfdca2" } 
+                ]
+        },
+        {
+            "name": "cuda",
+            "buildsystem": "simple",
+            "build-commands":["mkdir /app/tmp","sh cuda_12.2.1_535.86.10_linux.run --silent --toolkitpath=/app/opt/cuda --toolkit --override --tmpdir=/app/tmp"],
+            "sources":[
+                {
+                    "type": "file",
+                    "url": "https://developer.download.nvidia.com/compute/cuda/12.2.1/local_installers/cuda_12.2.1_535.86.10_linux.run",
+                    "sha256": "c51e814eaa756160a73289b1f35880cc38c18e1ddedc87f46f7ff0d1696bdb95"
+                }
+            ]
+        },
+        {
+            "name": "LightwaveExplorer",
+            "buildsystem": "cmake-ninja",
+            "config-opts": ["-DMAKECUDAFLATPAK=TRUE", "-DCMAKE_CUDA_ARCHITECTURES=75", "-DCUDAToolkit_ROOT=/app/opt/cuda","-DCMAKE_CUDA_COMPILER=/app/opt/cuda/bin/nvcc"],
+            "build-options": {
+                "cxxflags": "-O3 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong -grecord-gcc-switches",
+                "cxxflags-override": true},
```

Comment:
if the code triggers the assertion it should be fixed. disabling the assertions is just hiding the potential crashes.

Reply:
I modified the code and point the manifest to the new tagged version in commit 40eac10c4e2c8ed439b978be97cbfd37708f786e. Earlier, I would check if the result of a call to std::exp() was NaN, which would cause the assertion to fail for a bad argument; now I check the argument first. Simulation results are the same, but now I don't override the build options.